canonical-web-and-design/vanilla-framework
# Add discourse module to Vanilla docs

## Done

Connects Vanilla docs to Discourse Design System category (https://discourse.ubuntu.com/c/design-system/98)

Fixes canonical-web-and-design/vanilla-squad#1143

DEMO:
https://vanilla-framework-4353.demos.haus/design/patterns/segmented-control


## QA

- Open [demo](https://vanilla-framework-4353.demos.haus/design/patterns/segmented-control)
- Make sure Segmented control design guildelines appear as expected
- Compare with discourse source: https://discourse.ubuntu.com/t/segmented-control/26835

Note: This demo page is not meant to be currently linked from anywhere in Vanilla. This is just to make sure the connection between discourse category and Vanilla site works.


## Screenshots

<img width="1168" alt="Screenshot 2022-03-09 at 12 00 19" src="https://user-images.githubusercontent.com/83575/157428867-1edb2dab-a4b9-4cb4-a8cd-7c4ca83204c6.png">
